### ISSUE TITLE
CLI - Bad error message when maven is not present

### ISSUE THREAD

**Issue by romain-grecourt** [oracle/helidon-build-tools#304]
With only the JDK installed, the init command fails with a confusing error message.
The message should indicate that maven is not installed.

```
helidon init --debug

error: C:\Program Files\AdoptOpenJDK\jdk-11.0.8.10-hotspot\bin\mvn.cmd
java.lang.IllegalStateException: C:\Program Files\AdoptOpenJDK\jdk-11.0.8.10-hotspot\bin\mvn.cmd
        at io.helidon.build.util.MavenCommand.mavenExecutable(MavenCommand.java:129)
        at io.helidon.build.util.MavenCommand.mavenHome(MavenCommand.java:143)
        at io.helidon.build.util.MavenCommand.installedVersion(MavenCommand.java:156)
        at io.helidon.build.util.MavenCommand.assertRequiredMavenVersion(MavenCommand.java:196)
        at io.helidon.build.cli.impl.CommandRequirements.requireMinimumMavenVersion(CommandRequirements.java:89)
        at io.helidon.build.cli.impl.InitCommand.assertPreconditions(InitCommand.java:136)
        at io.helidon.build.cli.impl.BaseCommand.execute(BaseCommand.java:52)
        at io.helidon.build.cli.harness.CommandRunner.doExecuteCommand(CommandRunner.java:86)
        at java.util.Optional.ifPresentOrElse(Optional.java:201)
        at io.helidon.build.cli.harness.CommandRunner.doExecuteCommandName(CommandRunner.java:60)
        at java.util.Optional.ifPresentOrElse(Optional.java:201)
        at io.helidon.build.cli.harness.CommandRunner.doExecute(CommandRunner.java:50)
        at java.util.Optional.ifPresentOrElse(Optional.java:203)
        at io.helidon.build.cli.harness.CommandRunner.execute(CommandRunner.java:43)
        at io.helidon.build.cli.harness.CommandRunner.execute(CommandRunner.java:118)
        at io.helidon.build.cli.harness.CommandRunner.execute(CommandRunner.java:132)
        at io.helidon.build.cli.impl.Main.main(Main.java:36)
```

**Comment by romain-grecourt** [comment]
Fixed by #388

**Comment by romain-grecourt** [comment]
Needs backport to 2.x branch
